### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nobody5050**
~~please I need hacktoberfest~~

### CODE REVIEW

**Review by torralbaa — APPROVED** [review]
Sure, looks good, but I would wait for other people's reviews.

**Comment by nobody5050** [comment]
aight but if it's majority approved before hacktober ends i'll merge it

**Review by StealthHydra179 — APPROVED** [review]

**Comment by nobody5050** [comment]
told you so

**Pull request merged**
